scripts/globalSetup.js

```diff
@@ -1,3 +1,28 @@
 const { version } = require('webpack');
+// eslint-disable-next-line node/no-unpublished-require
+const rimraf = require('rimraf');
+const { join } = require('path');
+const collectTestFolders = require('./utils');
 
-module.exports = () => console.log(`\n Running tests for webpack @${version} \n`);
+// Cleanup obsolete snapshots for non-relevant webpack version
+const isWebpack5 = version[0] === '5';
+
+const snapshotDirs = ['help'].map(() => `__snapshots-v${isWebpack5 ? '4' : '5'}__`);
+
+const folderStrategy = (stats, file) => {
+    return stats.isDirectory() && snapshotDirs.includes(file);
+};
+
+const cleanupOutputDirs = () => {
+    for (const outputFolder of collectTestFolders(folderStrategy)) {
+        snapshotDirs.forEach((dir) => {
+            console.log(`removed ${join(outputFolder, dir)}`);
+            rimraf.sync(join(outputFolder, dir));
+        });
+    }
+};
+
+module.exports = () => {
+    cleanupOutputDirs();
+    console.log(`\n Running tests for webpack @${version} \n`);
+};
```

Comment:
Please clarify why we need remove something?

Reply:
no need removed